mmf/datasets/processors/frcnn_processor.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+"""
+ coding=utf-8
+ Copyright 2018, Antonio Mendoza Hao Tan, Mohit Bansal
+ Adapted From Facebook Inc, Detectron2
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.import copy
+ """
+import os
+import sys
+from io import BytesIO
+from typing import List
+
+import numpy as np
+import omegaconf
+import requests
+import torch
+import torch.nn.functional as F
+from mmf.common.registry import registry
+from mmf.datasets.processors.processors import BaseProcessor
+from PIL import Image
+
+
+class ResizeShortestEdge:
```

Comment:
We have this already in MMF. Please check if you can reuse: https://github.com/facebookresearch/mmf/blob/8409361b704cd2e90b410a71165a6073caa8f13a/tools/scripts/features/frcnn/processing_image.py#L28

Reply:
I will replace the usage of that preprocessor to use this